src/core/microservices/event-payloads/callout.published.event.payload.ts

```diff
@@ -0,0 +1,24 @@
+export type CalloutPublishedEventPayload = {
+  userID: string;
+  callout: {
+    id: string;
+    displayName: string;
```

Comment:
I would also add the description so that it is also available in the email template.